@types/jszip/index.d.ts

```
@@ -0,0 +1,346 @@
+     * @param data Serialized zip file
+     * @param options Options for deserializing
+     * @return Returns promise
+     */
+    loadAsync(data: InputFileFormat, options?: JSZip.JSZipLoadOptions): Promise<JSZip>;
+
+    /**
+     * Create JSZip instance
+     */
+    new(): this;
+
+    (): JSZip;
```

Biome (1.9.4)

[error] Don't use the new method in interfaces.

new in an interface suggests it's instantiable, which is incorrect. The returned type should different from the constructor's type.

(lint/suspicious/noMisleadingInstantiator)